packages/http/src/models/Balance.ts

```diff
@@ -22,12 +22,14 @@ export default class Balance {
   public floatBalance: number
   public type: CurrencyType
   public bigBalance: BigNumber
+  public height?: number
 
-  constructor(integerBalance: number | undefined, type: CurrencyType) {
+  constructor(integerBalance: number | undefined, type: CurrencyType, height?: number) {
```

Comment:
i wasn't sure the preferred order here so i just put it at the end

Reply:
Is it necessary to include `height` within Balance? I know the API returns `time` and `height` depending on the route so there's already a bit of inconsistency there. Maybe we could do like you do in the predicted price route and return a `{ price, height }` object?

Reply:
That would also have the advantage of conforming more closely with what the API returns

Reply:
i like that, i'll make the change there